grafana-plugin/src/utils/consts.ts

```diff
@@ -15,6 +15,9 @@ export const DEFAULT_PAGE = 'incidents';
 
 export const PLUGIN_ROOT = '/a/grafana-oncall-app';
 
+// https://developers.google.com/recaptcha/docs/v3
+export const reCAPTCHA_site_key = '6LeIPJ8kAAAAAJdUfjO3uUtQtVxsYf93y46mTec1';
```

Comment:
is there a way we can add constants like this into `process.env` at build time?

I'm a bit weary of adding constants like this because if we decide to add different recaptcha environments (ex. one for dev/staging/prod), how would we handle that here?

Reply:
you are right, let's discuss https://raintank-corp.slack.com/archives/C04JCU51NF8/p1677059171447329